README.md

```diff
@@ -13,5 +13,6 @@ fully featured Offline Workflow and Conflict Resolution.
 
 ## Example application
 
-See `sample` for example application
+See `sample` for example application. It uses voyager server as its backened (https://github.com/aerogear/voyager-server)
```

Comment:
You ment ionic showcase server?

Reply:
@wtrocki yeah, ionic showcase is based on voyager server so I added the link for the same

Reply:
I mean that we need to include server that is in showcase to even run the application. This information is missing. There is no need to use voyager-sever. Voyager-server provides additional capabilities but things will work with apollo graphql as well.